src/client.rs

```diff
@@ -0,0 +1,9 @@
+use crate::Utxo;
+use anyhow::Result;
+use async_trait::async_trait;
+use elements::{Address, TxOut};
+
+#[async_trait]
+pub trait GetTxOuts {
+    async fn get_txouts(&self, address: Address) -> Result<Vec<(Utxo, TxOut)>>;
+}
```

Comment:
This one is semantically owned by the wallet and should live next to it IMO.

Think about it this way: This is only useful as long as `wallet` exists. If you were to delete wallet, what else would you need to delete? If this lives next to wallet, deleting `wallet.rs` gets rid of everything that was introduced as part of it.